src/emulator/controller.ts

```diff
@@ -129,7 +129,7 @@ export async function startEmulator(instance: EmulatorInstance): Promise<void> {
   const name = instance.getName();
 
   // Log the command for analytics
-  track("emulators:start", name);
+  track("emulator:boot", name);
```

Comment:
```suggestion
  track("emulators:boot", name);
```

Reply:
nvm I thought it was a typo, now I see it is intentional

Reply:
So I thought about that, but this event is now only a single emulator starting hence `emulator` whereas `emulators:start` is several so it's plural? idk, I wasn't sure whats right. If you feel remotely strongly we can go plural :)

Reply:
I'd still suggest something unique that cannot be mistaken as a command -- something like `Emulator Run` (following the format of `Hosting Deploy`)